### ISSUE TITLE
using threejs on mobile

### ISSUE THREAD

**Issue by ibrahimAboelsuod** [mrdoob/three.js#7466]
In Android you can either use the Chrome app to show your webpage or use phonegap/cordova and used the embedded WebView.
Until Lollipop the default WebView depended on OS updates, now on devices from Lollipop (5.x) and up there is a 'Android System WebView' which is updated.
From this page: https://developer.chrome.com/multidevice/webview/overview You can see that WebGL support wasn't added until WebView v36.

-Long story short: threejs should work fine with phonegap on an a Lolipop device, right?
-This example  http://threejs.org/docs/index.html#Manual/Introduction/Creating_a_scene
works fine in the browser but just displays a blank page on device (both in app and device browser)
so it's not a phonegap problem as I think.

*Is there is different process used for mobiles other than the standard one in the example?
